### PR TITLE
[FIX] l10n_ar: inactivate argentinean doc types

### PR DESCRIPTION
**Pull request by bruno-zanotti**
**Description of the issue/feature this PR addresses:**
In Argentinean localization, we are making inactive some document types in a
no-updatable xml view to keep the user configuration when the module is
updated. This fix is to avoid the possibility of inactivating doc types
from other localizations with the same code.

Current behavior before PR:
If there is a doc type from another localization with the same code that we are inactivating it will be inactivated.

Desired behavior after PR is merged:
We will only inactivate Argentinean codes

> l10n_ar task 674

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_ar/data/l10n_latam.document.type.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
-            <value model="l10n_latam.document.type" eval="obj().search([('code', 'in', ['5','10','14','16','22','30','31','32','34','35','36','37','38','50','55','56','57','58','59','60','61','65','67','68','70','71','73','74','75','80','84','85','86','87','88','89','90','91','92','93','94','95','96','97','101','102','103','104','94','23','24','25','26','33','331','332','150','151','157','158','159','160','161','162','163','164','165','166','167','168','169','170','171','172','180','182','183','185','186','188','189','190','191'])]).ids"/>
+            <value model="l10n_latam.document.type" eval="obj().search([('country_id', '=', obj().env.ref('base.ar').id), ('code', 'in', ['5','10','14','16','22','30','31','32','34','35','36','37','38','50','55','56','57','58','59','60','61','65','67','68','70','71','73','74','75','80','84','85','86','87','88','89','90','91','92','93','94','95','96','97','101','102','103','104','94','23','24','25','26','33','331','332','150','151','157','158','159','160','161','162','163','164','165','166','167','168','169','170','171','172','180','182','183','185','186','188','189','190','191'])]).ids"/>
```
Comment (vbe-odoo) [inline addons/l10n_ar/data/l10n_latam.document.type.xml:5]: @Bruno-Zanotti use country_id.code

Comment (Bruno-Zanotti) [reply addons/l10n_ar/data/l10n_latam.document.type.xml]: @vbe-odoo done!

**Comment by vbe-odoo** [comment]
Hi @Bruno-Zanotti thanks for the FIX.
FUN Test done and LGTM. @jco-odoo ready for technical review.
Thanks

**Comment by jco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**